package.json

```diff
@@ -28,6 +28,7 @@
   },
   "homepage": "https://github.com/runk/node-geolite2#readme",
   "dependencies": {
+    "md5-file": "^4.0.0",
```

Comment:
Maybe it's a good idea to use a SHA-2 hash like SHA-384 instead of MD5? It's more future-proof and is recommended for resource Integrity by W3C, as in this spec: https://w3c.github.io/webappsec-subresource-integrity/#resource-integrity.

Reply:
Sure. I used md5 because that's what MaxMind use for their checksums but didn't think too much about it. I'l' switch to SHA-384

Reply:
62e752f